ztp/source-crs/ClusterVersionStatus.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: config.openshift.io/v1
+kind: ClusterVersion
+metadata:
+  name: version
+  annotations:
+    ran.openshift.io/ztp-deploy-wave: "100"
+status:
+  conditions:
+  - type: Available
+    status: "True"
+
```

Comment:
1. This condition will be True even if upgrade did not start at all.
2. How are we verifying the actual version?

Reply:
@yliu127, talked to Nishant and made a "quick" test on my dev env and it seems that we can just use the history status and skip this condition check. For example:
```
status:
    history:
    - state: Completed
      version: 4.9.8
```

Reply:
Yes agreed this could be the solution for 4.10. For 4.11, maybe we can ask if CVO can add a boolean status field as Ian suggested. e.g., 
status:
  atDesiredVersion: true